runbotics/runbotics-orchestrator-ui/src/utils/text.ts

```diff
@@ -1,7 +1,12 @@
-export const capitalizeFirstLetter = (text: string) => text.charAt(0).toUpperCase() + text.slice(1).toLowerCase();
+interface IText {
```

Comment:
I in front of the interfaces is an antipattern, name it more specifically like `CapitalizeFirstLetterParams`

Reply:
roger that. Done